tests/.sauce/config_on_sauce.yml

```
@@ -0,0 +1,15 @@
+apiVersion: v1alpha
+kind: puppeteer-replay
+sauce:
+  region: us-west-1
+  concurrency: 10
+suites:
+  - name: "getting some coffee"
+    platformName: "Windows 10"
```

Maybe we should set ourselves up with a good ol' leading by example and use the newest version of windows 💪

```suggestion
    platform: "Windows 11"
```